test/parallel/test-runner-mocking-fake-timers.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+process.env.NODE_TEST_KNOWN_GLOBALS = 0;
+const common = require('../common');
+
+const assert = require('node:assert');
+const { it, mock, afterEach, describe } = require('node:test');
+const nodeTimers = require('node:timers');
+const nodeTimersPromises = require('node:timers/promises');
+
+const { fakeTimers } = mock;
+
+describe('Faketimers Test Suite', () => {
```

Comment:
```suggestion
describe('Mock timers Test Suite', () => {
```

Reply:
These are not mocks, in the classic testing terminology, are they? We cannot instrument and change their behavior for later verification. These are more like fakes, with working implementations. Or are you using "mock" in the colloquial sense where it is used for anything that stubs out some existing functionality?